server/models/assessRisksAndNeeds/riskSummary.ts

```diff
@@ -24,5 +24,5 @@ export default interface RiskSummary {
     riskMitigationFactors?: string | null
     riskInCommunity: Partial<Record<RiskScore, string[]>>
   }
-  assessedOn: string
+  assessedOn?: string | null
```

Comment:
My understanding of the ARN contract is that this is never null?
Curious if you found it not to be the case?

Reply:
We were following the example with all this possibly being undefined above - do you think it's safe to rely on the field? and when you say this is never null, do you mean never `null` _or_ `undefined`?

Reply:
Ah I see that now - it's nice to have a clean contract...

... but I suppose based on previous experience with communityApi what they assert is not always the reality.

Reply:
As discussed on slack, it looks like this could come back empty as it comes from the assessments API (not ARN) and could be undefined:

https://github.com/ministryofjustice/hmpps-assess-risks-and-needs/blob/cd927d4bda94ac24ca45332ee752d3dcc5119823/src/main/kotlin/uk/gov/justice/digital/hmpps/assessrisksandneeds/services/RiskService.kt#L31